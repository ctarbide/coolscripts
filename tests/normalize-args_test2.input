CHMOD='chmod 0555' nofake.sh --error -Rnofake-exec.sh -onofake-exec.sh shell.nw temporary-files-shell.nw nofake-exec.nw && ./nofake-exec.sh -R"a 'b c'" nofake-exec.nw -- cat
