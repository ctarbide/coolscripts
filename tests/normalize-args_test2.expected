'CHMOD=chmod 0555'
'nofake.sh'
'--error'
'-Rnofake-exec.sh'
'-onofake-exec.sh'
'shell.nw'
'temporary-files-shell.nw'
'nofake-exec.nw'
'&&'
'./nofake-exec.sh'
'-Ra '"'"'b c'"'"
'nofake-exec.nw'
'--'
'cat'
